src/software/thunderscope/robot_communication.py

```diff
@@ -219,53 +238,68 @@ def __run_primitive_set(self) -> None:
         is useful to dip in and out of robot diagnostics.
         """
         while self.running:
-            # total primitives for all robots
-            robot_primitives = {}
+            # map of robot id to diagnostics/fullsystem primitive map
```

Comment:
__run_primitive_set's docs should be updated with details about handheld controller forwarding. Also looks like these docs are now outdated